sphinx/pycode/parser.py

```diff
@@ -396,6 +396,15 @@ def visit_Assign(self, node: ast.Assign) -> None:
 
         # not commented (record deforders only)
         for varname in varnames:
+            if (
+                # TODO(picnixz): decide whether to include special names or not
```

Comment:
I am considering possibly picking attributes defined in `__new__` as well but I think it should be another PR. I don't remember why I used "special" names though...